cura/PrinterOutput/PrinterOutputModel.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, output_controller: "PrinterOutputController", number_of_extru
         # Update the printer configuration every time any of the extruders changes its configuration
         for extruder in self._extruders:
             extruder.extruderConfigurationChanged.connect(self._updatePrinterConfiguration)
+        self.typeChanged.connect(self._updatePrinterConfiguration)
```

Comment:
Maybe use self.machineTypeChanged? Makes it a bit more explicity what type we're talking about.

Reply:
I've just used a previously created signal. I agree in changing the name. Done in 54882402adcf00a7cd6a844173d34ef975f9656b